settings.py

```diff
@@ -1277,6 +1277,8 @@
 
 SYNCLOGS_DB = 'synclogs'
 
+META_DB = 'meta'
+
```

Comment:
Is this related to the other commit or did it just kind of sneak its way in here?